util/characteristics.js

```diff
@@ -434,5 +440,42 @@ module.exports = function (Characteristic, units)
 	inherits(CustomCharacteristic.WindSpeedMax, Characteristic);
 	CustomCharacteristic.WindSpeedMax._unitvalue = windspeedValue;
 
+    CustomCharacteristic.WindSpeedLull = function () {
+        Characteristic.call(this, 'Wind Speed Lull', CustomUUID.WindSpeedLull);
+        this.setProps(windspeedProps(100));
+        this.value = this.getDefaultValue();
+    };
+    inherits(CustomCharacteristic.WindSpeedLull, Characteristic);
+    CustomCharacteristic.WindSpeedLull._unitvalue = windspeedValue;
+    
+	CustomCharacteristic.LightningStrikes = function () {
+        Characteristic.call(this, 'Lightning Strikes', CustomUUID.LightningStrikes);
+        this.setProps({
+            format: Characteristic.Formats.UINT8,
+            maxValue: 1000,
+            minValue: 0,
+            minStep: 1,
+            perms: [Characteristic.Perms.READ, Characteristic.Perms.NOTIFY]
+        });
+        this.value = this.getDefaultValue();
+    };
+    inherits(CustomCharacteristic.LightningStrikes, Characteristic);
+    
+    CustomCharacteristic.LightningAvgDistance = function () {
+        Characteristic.call(this, 'Lightning Avg Distance', CustomUUID.LightningAvgDistance);
+        this.setProps(visibilityProps(100));
+        this.value = this.getDefaultValue();
+    };
+    inherits(CustomCharacteristic.LightningAvgDistance, Characteristic);
+    CustomCharacteristic.LightningAvgDistance._unitvalue = visibilityValue;
+
+    CustomCharacteristic.TemperatureWetBulb = function ()
+	{
+		Characteristic.call(this, 'Wet-bulb temperature', CustomUUID.TemperatureWetBulb);
+		this.setProps(temperatureProps(-50, 100));
```

Comment:
Based on https://en.wikipedia.org/wiki/Wet-bulb_temperature#Highest_recorded_wet-bulb_temperatures, `100` seems a very high maximum 😅 I'd think that `40` would be enough.

So maybe:
```suggestion
                // @see https://en.wikipedia.org/wiki/Wet-bulb_temperature
		this.setProps(temperatureProps(-50, 50));
```
?

Reply:
Wet-bulb temperature is just another 'temperature' value, so I was basing the numbers on the TemperatureMin and TemperatureApparent Characteristic defined in the same file. I can change this to be something more custom, should we really do that though?

Reply:
What you define here is what apps will present you as the range of values to select in automations. So IMHO: yes.